src/components/sbb-icon/sbb-icon.scss

```diff
@@ -3,31 +3,33 @@
 @include host-component-properties;
 
 :host {
-  --sbb-icon-margin-inline: 0;
-  --sbb-icon-margin-block: 0;
-
   display: inline-block;
   line-height: 0;
-  margin-inline: var(--sbb-icon-margin-inline-override, var(--sbb-icon-margin-inline));
-  margin-block: var(--sbb-icon-margin-block-override, var(--sbb-icon-margin-block));
+  margin-inline: var(--sbb-icon-margin-inline-override);
+  margin-block: var(--sbb-icon-margin-block-override);
 }
 
-svg:not(.color-immutable) {
-  [fill]:not([fill='none']) {
-    fill: currentcolor;
-  }
-
-  [stroke]:not([stroke='none']) {
-    stroke: currentcolor;
-  }
+svg {
+  width: var(--sbb-icon-width-override);
```

Comment:
I'm a bit torn on naming here. (Also applies to height.)
What do you think about `--sbb-icon-svg-width`?
My thought is; Since we don't define a value ourselves, it's not really an override.
What do you think?

Reply:
Exactly, I would have raised the same question in the next refinement. I agree with you for the most part, then I thought maybe we wanted to have the same naming convention to express the concept of "overriding the default value/behaviour", which in this case would be: "the size is defined by the retrieved icon, but you can override it". But I would probably change it to `--sbb-icon-svg-width` as you suggest anyway...